config/src/main/resources/dev.json

```diff
@@ -1,6 +1,6 @@
 {
   "config": {
-    "chainId": 2018,
+    "chainId": 1,
```

Comment:
not sure what number this should be - do we want to avoid low numbers due to conflicts with existing chains? AJS would be smarter than I...